### PR TITLE
add azure due to missing commit and update submodules

### PR DESCRIPTION
**Pull request by yaojiefeng**

### CODE REVIEW

**Review by elfenheart — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -35,3 +35,7 @@
 	path = condition-plugins
 	url = ../../data-integrations/condition-plugins.git
 	branch = develop
+[submodule "azure"]
+	path = azure
+	url = https://github.com/data-integrations/azure
```
Comment (MEseifan) [inline .gitmodules]: Maybe change this to `../../data-integrations/azure` to match the format for the other URLs?

Comment (yaojiefeng) [reply .gitmodules]: Fixed

**Pull request merged**